src/open_inwoner/pdc/admin/product.py

```diff
@@ -40,6 +42,41 @@ class Meta:
         fields = "__all__"
         widgets = {"content": CKEditorWidget}
 
+    categories = forms.ModelMultipleChoiceField(
```

Comment:
Should I be able to see/edit this field if I'm logged in as a user that has managed categories? I was able to remove categories from a product here and block my own access to this product after doing so

Reply:
I added a check in the form and a test that forces there to always have at least one category.